### ISSUE TITLE
Return number

### ISSUE THREAD

**Issue by msmanui** [Andre601/Math-Expansion#11]
### I confirm that...

- [X] I use the latest version of the expansion. [[Expansion Page](https://api.extendedclip.com/expansions/math/) | [Changelog](https://github.com/Andre601/Math-Expansion/blob/master/CHANGELOG.md)]
- [X] The issue is not caused by PlaceholderAPI or any other plugin or expansion
- [X] Any used placeholder works as intendet (`/papi parse me <placeholders>`)

### Description

Okay so, im trying to divide two placeholders that both equals 0, the result is that the placeholder wont parse.
My solution would be in case the divisor equals to "0", then return the divider. 
So if the player has 0 kills and 0 death, it would return 0, if its has 6/0 then 6.

`%math_2:2_{statistic_player_kills}/{statistic_deaths}%`

### Logs

```text
No need.
```

### Math Expansion version

```text
1.3.2
```

### PlaceholderAPI version

```text
2.10.10-DEV-122
```

### Server version

```text
1.17.1
```

**Comment by Andre601** [comment]
I would love to apply a fix, but there are several problems here:

- Math Expansion doesn't do all the calculations itself. It simply parses placeholders and then sends the result to EvalEx to handle.
- The expansion doesn't know when any calculation would contain 0s and also can't really know when a calculation would be "illegal" (i.e. divide by 0).
- Implementing a system that checks for this would be difficult and pretty much impossible, as you would need to count for other calculations that are valid when using 0s (subtraction and addition f.e.), not to mention would I also need to look for the general logic (multiplication/division before addition/subtraction) in larger calculations which may or may not contain not-allowed expressions.

tl;dr: This is nothing I can fix on my end and you're better of checking if EvalEx would have a solution/workaround for this.
